content/en/contribute.md

```diff
@@ -105,10 +98,27 @@ sprints](https://scisprints.github.io/), a newsletter, and perhaps a blog.
 
 ### Fundraising
 
-NumPy was all-volunteer for many years, but as its importance grew it became
-clear that to ensure stability and growth we'd need financial support. [This
-SciPy'19 talk](https://www.youtube.com/watch?v=dBTJD_FDVjU) explains how much
-difference that support has made. Like all the nonprofit world, we're
-constantly searching for grants, sponsorships, and other kinds of support. We
-have a number of ideas and of course we welcome more. Fundraising is a scarce
-skill here -- we'd appreciate your help.
+For many years, NumPy was maintained by dedicated volunteers, but as its importance grew it 
+became clear that to ensure stability and growth we would need financial support.
+[This SciPy'19 talk](https://www.youtube.com/watch?v=dBTJD_FDVjU) explains how much difference 
+that support has made. Like most nonprofits, we are constantly seeking grants, sponsorships, 
+and other kinds of funding. We have a number of ideas and of course we welcome more. 
+Fundraising is a scarce skill here -- we'd appreciate your help.
+
+### Donate
+
+If you'd like to contribute to NumPy by making a donation, you are welcome to do so. 
```

Comment:
It would be better to have one sentence here linking to `about.md#donate` (see https://numpy.org/about/#donate), rather than duplicate this content.

Reply:
That content on the About page also needs a small update, to point to Open Collective (as pointed out in https://github.com/numpy/numpy.org/issues/498#issuecomment-1295930356)

Reply:
@rgommers Thank you for your feedback! The button in `about.md#donate` is currently pointing to https://opencollective.com/numpy#category-CONTRIBUTE Did you have a different "destination" in mind?

Reply:
Sorry for the noise, the link is fine. I was either looking at an old page or had some temporary blindness - that comment was addressed a long time ago. Please ignore this part.